src/asyncresults.jl

```diff
@@ -31,7 +31,7 @@ function Base.show(io::IO, async_result::AsyncResult)
     else
         "in progress"
     end
-    print(io, typeof(async_result), " (", status, ")")
+    return print(io, typeof(async_result), " (", status, ")")
```

Comment:
```suggestion
    print(io, typeof(async_result), " (", status, ")")
    return nothing
```

Should `Base.show` always return `nothing`?

Reply:
yeah, i think your suggestion is good but the code in this PR is also fine?

Reply:
The latter change would require checking if the function returns anything though which is not a syntactic analysis afaik and may be outside the scope of the formatter, at least for now.